packages/ihe-gateway-sdk/src/models/xca_iti_38.ts

```diff
@@ -0,0 +1,34 @@
+import { baseResponseSchema, documentReference, baseRequestSchema } from "./shared";
+import { z } from "zod";
+
+export const code = z.object({
+  system: z.string(),
+  code: z.string(),
+});
+
+export const dateRange = z.object({
+  dateFrom: z.string(),
+  dateTo: z.string(),
+});
+
+export const xcaIti38RequestSchema = z.array(
+  baseRequestSchema.extend({
+    homeCommunityId: z.string(),
+    xcpdPatientId: z.object({ id: z.string(), system: z.string() }),
+    patientResourceId: z.string().nullable(),
+    xcaGateway: z.string(),
+    classCode: z.array(code).nullable(),
+    practiceSettingCode: z.array(code).nullable(),
+    facilityTypeCode: z.array(code).nullable(),
+    documentCreationDate: dateRange.nullable(),
+    serviceDate: dateRange.nullable(),
+  })
+);
+
+export type XCA_ITI_38Request = z.infer<typeof xcaIti38RequestSchema>;
```

Comment:
Can we use business names for these types, please?

In the context of this SDK, there's a 1:1 relationship between `DocumentQuery` and `XCA_ITI_38Request`, so no need to reference the protocol used to implement the doc query, it only makes it harder to understand and type. We can have the protocol on the function's TSDoc.

For more context: https://metriport.slack.com/archives/C065BLRBUDQ/p1701314896551959?thread_ts=1701037512.758309&cid=C065BLRBUDQ